docs/topics/DatabaseOperations.adoc

```diff
@@ -61,7 +61,7 @@ image::database_view.png[]
 === Quick Unlock
 On Windows and macOS, subject to hardware availability, your credentials can be securely stored to enable subsequent unlocking of your database through biometric authentication. This is enabled by default on Windows using _Windows Hello_ and on macOS using _Touch ID or Apple Watch_ services. You can disable this feature in the Application Settings under the Security section.
 
-NOTE: On Windows you will be prompted to authenticate to Windows Hello on the initial database unlock. This is required to access the hardware certificate store that encrypts your credentials.
+NOTE: On Windows, you will be prompted to authenticate to Windows Hello after unlocking your database with full credentials. This is required to access the hardware certificate store that secures Quick Unlock. If you cancel this prompt, your database will continue to unlock, but Quick Unlock will not be enabled.

```

Comment:
I think this would be clearer:
```suggestion
NOTE: On Windows, you will be prompted to authenticate to Windows Hello after unlocking your database with full credentials. This is required to store your credentials in a secure hardware store. If you cancel this prompt, your database will continue to unlock, but Quick Unlock will not be enabled.
```

Reply:
However, it is completely false. We don't store credentials in a hardware store.

Reply:
You store _something_ in there, don't you? (Encrypted credentials? Some identifier? I don't remember the details, but that's less important). Otherwise you wouldn't need that prompt.

Anyway, maybe just "This is required to configure QuickUnlock."

Reply:
And then those encrypted credentials still need to be stored somewhere? So that they would be available to unlock the database later.

Anyway, those technical details are besides the point. I still think that "access the hardware certificate store that secures Quick Unlock" is too technical.

Reply:
Yah the encrypted credentials are stored in keepassxc memory.

Reply:
Looks good.